### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
Please review

### CODE REVIEW

**Review by kushalsourav — APPROVED** [review]
looks good to merge,  
suggestion --  for btn-quiz you could hav used component library button itself , your choice.

**Comment by shruthipatil11** [comment]
UI is cool . I would recommend using BEM for naming convention to improve readability but yes its up to developer choice.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://canvas-component-lib.netlify.app/Component.css">
+    <title>Document|Home</title>
+</head>
+<body>
+    <div class="desktop-container">
+      <img src="https://hips.hearstapps.com/hmg-prod.s3.amazonaws.com/images/nyfwsketches-1567608885.jpg?crop=1.00xw:1.00xh;0,0&resize=980:*" alt="Fashion-backdrop" class="responsive-img quiz-background-img" />
```
Comment (SinchanaMS) [inline index.html]: The code looks clean. 
For the UI, I'd suggest you decrease the opacity of the bg image for better visibility of the button.

**Review by SinchanaMS — APPROVED** [review]

File: style.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital@1&display=swap');
+:root
+{
+    --button-color: palevioletred;
```
Comment (ashishkatiyar2404) [inline style.css]: try to use RGB color in root

**Review by ashishkatiyar2404 — APPROVED** [review]
your code looks good to go.🚀

File: index.html (hunk 1 above)
Comment (ShrutiLakhani) [reply index.html]: Done

File: style.css (hunk 2 above)
Comment (ShrutiLakhani) [reply style.css]: Done

**Pull request merged**